db/migrate/20210521192754_update_product_features_where_identifier_was_renamed_since_lasker.rb

```diff
@@ -0,0 +1,26 @@
+class UpdateProductFeaturesWhereIdentifierWasRenamedSinceLasker < ActiveRecord::Migration[6.0]
+  class MiqProductFeature < ActiveRecord::Base; end
+  class MiqRolesFeature < ActiveRecord::Base; end
```

Comment:
When the stub is only needed in tests, we do it a little differently, and don't create the stub in the main code...can't recall how offhand, but you can search other specs.